builtin/providers/aws/resource_aws_instance.go

```diff
@@ -129,9 +129,11 @@ func resourceAwsInstance() *schema.Resource {
 			},
 
 			"network_interface_id": &schema.Schema{
-				Type:     schema.TypeString,
-				Computed: true,
-				Optional: true,
+
+				Type:          schema.TypeString,
```

Comment:
good question. idk anything about TypeSets and TypeLists yet. do you think that the initial implementation should focus on 1 or a list?